newsletter-june-2021.rst

```diff
@@ -0,0 +1,87 @@
+.. post:: June 3, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from June.
+
+Read the Docs newsletter - June 2021
+====================================
+
+Welcome to a new edition of our monthly newsletter, where we
+openly share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
```

Comment:
Do we still need this intro? Should we update it or delete it?

Reply:
There will always be people that read the newsletter for the first time, right? :)